src/inc/eetwain.h

```diff
@@ -161,7 +161,7 @@ enum
 };
 
 #ifndef DACCESS_COMPILE
-
+#ifdef _TARGET_X86_
```

Comment:
Correct. It is only called in src/vm/i386/excepx86.cpp that is only included for !WIN64EXCEPTIONS. You should not need it for Linux/x86.